en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -122,12 +122,17 @@ To configure logging in MI instances, you can use environment variables with spe
 2. Replace the dot character in the package name with an underscore.
 3. Set the variable value to the required logging level for the corresponding package or class.
 
-For example, to set the logging level for the `org.apache.synapse.transport.http.wire` package to `debug`, define the
-following environment variable:
-
-```
-export logging_level_org_apache_synapse_transport_http_wire=debug
-```
+For example, to enable wire logs logging level of the `org.apache.synapse.transport.http.wire` package needs to be changed
```

Comment:
For example, to enable wire logs, the logging level of the org.apache.synapse.transport.http.wire package must be adjusted to debug. In order to configure the logging, set the environment variable in the following manner: